packages/story-editor/src/testUtils/firePointerEvent.js

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { fireEvent } from '@testing-library/react';
+
+function firePointerEvent(node, eventType, properties) {
+  fireEvent(node, new window.MouseEvent(eventType, properties));
+}
+
+const pointerEventTypes = [
+  'pointerOver',
+  'pointerEnter',
+  'pointerDown',
+  'pointerMove',
+  'pointerUp',
+  'pointerCancel',
+  'pointerOut',
+  'pointerLeave',
+  'gotPointerCapture',
+  'lostPointerCapture',
+];
+
+pointerEventTypes.forEach((type) => {
+  firePointerEvent[type] = (node, properties) =>
+    firePointerEvent(node, type.toLowerCase(), {
+      bubbles: true,
+      ...properties,
+    });
+});
+
+export default firePointerEvent;
```

Comment:
Is this still needed?

How is this different to e.g. `fireEvent.pointerDown()` (which is a thing!)?

Reply:
`fireEvent.pointerDown`relies on the underlying DOM implementation to have a PointerEvent class, and JSDO still doesn't, as far as I can tell. I tried going directly through that event, but it doesn't work.

Maybe we need to upgrade JSDOM though, didn't test that out.